serverless.yml

```diff
@@ -0,0 +1,126 @@
+# Welcome to Serverless!
+#
+# This file is the main config file for your service.
+# It's very minimal at this point and uses default values.
+# You can always add more config options for more control.
+# We've included some commented out config examples here.
+# Just uncomment any of them to get that config option.
+#
+# For full config options, check the docs:
+#    docs.serverless.com
+#
+# Happy Coding!
+
+service: nedejs-aws-fe
+# app and org for use with dashboard.serverless.com
+#app: your-app-name
+#org: your-org-name
+
+# You can pin your service to only deploy with a specific Serverless version
+# Check out our docs for more details
+frameworkVersion: '2'
+
+plugins:
+  - serverless-finch
+  - serverless-single-page-app-plugin
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  profile: njsprogram
+  stage: production
```

Comment:
What does it mean this parameter?

Reply:
This exists for CI/CD pipelines of different developments environments (dev, int, uat, prod). Add prefix to CloudFormation stack and its related AWS resources.
https://www.serverless.com/blog/stages-and-environments